frontend/UserList.vue

```diff
@@ -0,0 +1,298 @@
+        </tr>
+      </thead>
+      <tbody id="user-list-body">
+        <tr v-for="(user, index) in users">
+          <td><button class="btn btn-default glyphicon glyphicon-pencil" @click="showModal('edit-user-modal', user, index)"></button></td>
+          <td>
+            <dropdown>
+              <button data-role="trigger" class="btn btn-default dropdown-toggle glyphicon glyphicon-option-horizontal" type="button">
+              </button>
+            </dropdown>
+          </td>
+          <td>{{user.email}}</td>
+          <td>{{user.admin}}</td>
+          <td>{{user.disabled}}</td>
+        </tr>
+      </tbody>
+    </table>
+    <modal
+       name="edit-user-modal"
+       :height="830"
+       classes="no-background-color"
+       @opened="modalOpened"
+       @closed="modalClosed"
+       >
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h2 class="modal-title" v-if="currentUser.id">Edit user</h2>
+            <h2 class="modal-title" v-if="!currentUser.id">New user</h2>
+            <button type="button" v-if="currentUser.id" class="pull-right btn btn-danger" @click="removeUser">Delete</button>
+          </div>
+          <div class="modal-body">
+            <form action="" id="editUserForm">
+              <p><label for="email">Email: </label><input class="form-control" type="text" v-model.trim="currentUser.email" id="email" /></p>
+              <p><label for="admin">Admin: </label><input class="form-control" type="checkbox" v-model="currentUser.admin" id="admin" /></p>
+              <p><label for="disabled">Disabled: </label><input class="form-control" type="checkbox" v-model="currentUser.disabled" id="disabled" /></p>
+            </form>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-secondary" @click="hideModal">Close</button>
+            <button type="button" v-bind:disabled="disableConfirmButton" class="btn btn-success" @click="confirmEditUserModal">Save changes</button>
+          </div>
+        </div>
+      </div>
+    </modal>
+  </div>
+
+</template>
+
+<script>
+// Library from here: https://github.com/euvl/vue-js-modal
+import vmodal from 'vue-js-modal';
+import Vue from 'vue';
+import {flashMessage} from 'flash_message';
+import {Dropdown} from 'uiv';
+
+Vue.use(vmodal);
+
+// Constants related to list ordering
+// Corresponds to the constants DescOrder and AscOrder in model/activist.go
+const DescOrder = 2;
+const AscOrder = 1;
+
+export default {
+  name: 'user-list',
+  methods: {
+    showModal: function (modalName, user, index) {
+      // Check to see if there's a modal open, and close it if so.
+      if (this.currentModalName) {
+        this.hideModal();
+      }
+
+      // Make shallow copy of selected activist to prevent persisting unsaved
+      // edits at the view layer when closing modal
+      if (user) {
+        this.currentUser = $.extend({}, user);
```

Comment:
`this.currentUser = $.extend({}, user);` works even if `user` doesn't exist

Reply:
Good catch. I'll change that.